src/main/resources/fabric.mod.json

```diff
@@ -31,7 +31,7 @@
         ]
     },
     "depends": {
-        "minecraft": ">=1.15.2 <1.16.0",
+        "minecraft": ">=1.16-pre2",
```

Comment:
hm this should actually either be `>=1.16-rc2` or just `1.16.x`, I just realized.